src/interface/cholesky.cpp

```diff
@@ -0,0 +1,31 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#include <dlaf/interface/cholesky.h>
+
+namespace dlaf::interface {
+
+extern "C" void pdpotrf(char uplo, int n, double* a, int ia, int ja, int* desca, int& info) {
```

Comment:
It's probably enough to put `extern "C"` on the definition, right? Assuming these are all lifted out of the `dlaf::interface` namespace.

Reply:
Yes, I think so.